documentation/meta/media_properties/api.md

```diff
@@ -0,0 +1,280 @@
+# API Media Properties
+
+_This document is auto-generated from the source code in
+[/api/api/management/commands/documentmedia.py](https://github.com/WordPress/openverse/blob/main/api/api/management/commands/documentmedia.py)._
+
+This is a list of the media properties, with the descriptions of corresponding
+database columns and Python objects that are used to store and retrieve media
+data.
+
+## General notes
+
+The columns are sorted alphabetically and separated into relations (used to
+establish relationships to other models) and values (used to hold some data
+value). Note that relation fields are always nullable.
+
+## Audio
+
+| Name              | Type              | DB type | Nature       | To               |
+| ----------------- | ----------------- | ------- | ------------ | ---------------- |
+| `audio_report`    | `ForeignKey`      | `uuid`  | One To Many  | `AudioReport`    |
+| `audioset`        | `ForeignObject`   |         | Many To One  | `AudioSet`       |
+| `deleted_audio`   | `OneToOneField`   | `uuid`  | One To One   | `DeletedAudio`   |
+| `lists`           | `ManyToManyField` |         | Many To Many | `AudioList`      |
+| `sensitive_audio` | `OneToOneField`   | `uuid`  | One To One   | `SensitiveAudio` |
+
+| Name                           | Type            | DB type                    | Constraints                   | Default |
+| ------------------------------ | --------------- | -------------------------- | ----------------------------- | ------- |
+| `alt_files`                    | `JSONField`     | `jsonb`                    |                               |         |
```

Comment:
I implemented the _absurdly cool_ portion of your comment in 4bbdf40. But the part about linking below doesn't work for all fields because Sphinx, for some reason, assigns some of them numeric IDs like `id<number>`.

<img width="855" alt="Screenshot 2024-04-05 at 11 53 27 AM" src="https://github.com/WordPress/openverse/assets/16580576/c9fff0e8-49c4-4eed-b210-12bb0fd3c252">

Reply:
Ok I worked around that issue. Internal hyperlinks added in e3d5176.